src/mono_font/mono_text_style.rs

```diff
@@ -203,15 +203,20 @@ where
         Ok(position + Size::new(width, 0))
     }
 
-    fn measure_text(&self, text: &str, position: Point) -> TextMetrics {
-        // TODO: ignore control characters in `text`
-
+    fn measure_string(&self, text: &str, position: Point) -> TextMetrics {
         let width = (text.len() as u32 * (F::CHARACTER_SIZE.width + F::CHARACTER_SPACING))
             .saturating_sub(F::CHARACTER_SPACING);
         let size = Size::new(width, F::CHARACTER_SIZE.height);
 
+        // Return a zero sized bounding box is the text is completely transparent.
```

Comment:
The typo ("box is the text is ..."), sounds like slang from Mistborn, lol, I approve :D

Reply:
Fixed.